### PR TITLE
Add FAQ #schnelltesttest

### PR DESCRIPTION
**Pull request by larswmh**
image
image

---
Internal Tracking ID: [EXPOSUREAPP-12146](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-12146)

### CODE REVIEW

File: src/data/faq_de.json [hunk 1]
```diff
@@ -971,6 +971,16 @@
...
                         ]
+                    },
+                    {
+                        "title": "Wie effizient erkennt der von mir verwendete Test eine Infektion?",
+                        "anchor": "schnelltesttest",
+                        "active": false,
+                        "textblock": [
+                            "<a href='https://schnelltesttest.de/' target='_blank' rel='noopener noreferrer'>schnelltesttest.de</a> ist ein Open Source Projekt welches dir einen einfachen Weg bietet, Informationen über die vom Paul-Ehrlich-Institut evaluierten SARS-COV-2-Antigen-Schnelltests abzurufen.",
```
Comment (Ein-Tim) [inline src/data/faq_de.json:980]:
```suggestion
                            "<a href='https://schnelltesttest.de/' target='_blank' rel='noopener noreferrer'>schnelltesttest.de</a> ist ein deutsches Open Source Projekt welches dir einen einfachen Weg bietet, Informationen über die vom Paul-Ehrlich-Institut evaluierten SARS-COV-2-Antigen-Schnelltests abzurufen.",
```

**Review by Ein-Tim — CHANGES_REQUESTED** [review]
Add "deutsches" to match English text.

File: src/data/faq_de.json (hunk 1 above)
Comment (MikeMcC399) [inline src/data/faq_de.json:980]: "welches dir" => "welches Ihnen"

to be consistent with the rest of the article and the FAQs in general.

File: src/data/faq_de.json [hunk 2]
```diff
@@ -971,6 +971,16 @@
...
                         ]
+                    },
+                    {
+                        "title": "Wie effizient erkennt der von mir verwendete Test eine Infektion?",
+                        "anchor": "schnelltesttest",
+                        "active": false,
+                        "textblock": [
+                            "<a href='https://schnelltesttest.de/' target='_blank' rel='noopener noreferrer'>schnelltesttest.de</a> ist ein Open Source Projekt welches dir einen einfachen Weg bietet, Informationen über die vom Paul-Ehrlich-Institut evaluierten SARS-COV-2-Antigen-Schnelltests abzurufen.",
+                            "Nach Angabe eines Tests, welche unter anderem durch einen Scan des Barcodes oder der Eingabe des Herstellers erfolgen kann, wird Ihnen angezeigt zu wie viel Prozent der jeweilige Test Infizierte mit sehr hoher Viruslast erkennt, und zu wie viel Prozent er über alle Viruslasten Infizierte erkennt.",
+                            "Es handelt sich hierbei um ein externes Projekt, welches in keinem Zusammenhang mit der Corona-Warn-App steht. Die Corona-Warn-App ist nicht für den Inhalt der genannten Seite verantwortlich."
```
Comment (MikeMcC399) [inline src/data/faq_de.json:982]: "Die Corona-Warn-App ist nicht für den Inhalt der genannten Seite verantwortlich."

how would the app itself CWA be responsible anyway?

Maybe "Der Herausgeber der Corona-Warn-App ist nicht für den Inhalt der genannten Seite verantwortlich."?

File: src/data/faq.json [hunk 3]
```diff
@@ -972,6 +972,16 @@
...
                         ]
+                    },
+                    {
+                        "title": "How efficiently does the test I use detect an infection?",
+                        "anchor": "schnelltesttest",
+                        "active": false,
+                        "textblock": [
+                            "<a href='https://schnelltesttest.de/' target='_blank' rel='noopener noreferrer'>schnelltesttest.de</a> is a German open source project that provides you with an easy way to retrieve information about the SARS-COV-2 antigen rapid tests evaluated by the Paul Ehrlich Institute.",
+                            "After specifying a test, which can be done by scanning the barcode or entering the manufacturer's name, you will be shown the percentage of infected persons with very high viral loads and the percentage of infected persons with all viral loads.",
+                            "This is an external project, which is not related to the Corona-Warn-App. The Corona-Warn-App is not responsible for the content of the mentioned page."
```
Comment (MikeMcC399) [inline src/data/faq.json:983]: "The Corona-Warn-App is not responsible for the content of the mentioned page." => 

"The publisher of the Corona-Warn-App is not responsible for the content of the mentioned page."

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
Various comments left.

File: src/data/faq_de.json (hunk 1 above)
Comment (larswmh) [reply src/data/faq_de.json]: Thanks for your suggested change!

File: src/data/faq.json (hunk 3 above)
Comment (larswmh) [reply src/data/faq.json]: Changed it in https://github.com/corona-warn-app/cwa-website/pull/2533/commits/3796aeb0d7637de624daf0310c8200a95d0dae96, thanks for your suggestion!

File: src/data/faq_de.json (hunk 2 above)
Comment (larswmh) [reply src/data/faq_de.json]: Changed it in https://github.com/corona-warn-app/cwa-website/pull/2533/commits/3796aeb0d7637de624daf0310c8200a95d0dae96, thanks for your suggestion!

File: src/data/faq.json [hunk 4]
```diff
@@ -972,6 +972,16 @@
...
                         ]
+                    },
+                    {
+                        "title": "How efficiently does the test I use detect an infection?",
+                        "anchor": "schnelltesttest",
+                        "active": false,
+                        "textblock": [
+                            "<a href='https://schnelltesttest.de/' target='_blank' rel='noopener noreferrer'>schnelltesttest.de</a> is a German open source project that provides you with an easy way to retrieve information about the SARS-COV-2 antigen rapid tests evaluated by the Paul Ehrlich Institute.",
+                            "After specifying a test, which can be done by scanning the barcode or entering the manufacturer's name, you will be shown the percentage of infected persons with very high viral loads and the percentage of infected persons with all viral loads.",
```
[the remark this answers is not in the document]
Comment (larswmh) [reply src/data/faq.json]: Added it to the end of the sentence in https://github.com/corona-warn-app/cwa-website/pull/2533/commits/3796aeb0d7637de624daf0310c8200a95d0dae96, thanks for your suggestion!

File: src/data/faq_de.json (hunk 1 above)
Comment (larswmh) [reply src/data/faq_de.json]: Changed it in https://github.com/corona-warn-app/cwa-website/pull/2533/commits/3796aeb0d7637de624daf0310c8200a95d0dae96, thanks for your suggestion!

**Comment by dsarkar** [comment]
@Ein-Tim @MikeMcC399 Again, many thanks for reviewing!

**Review by dsarkar — APPROVED** [review]

**Pull request merged**